tests/unit_tests/unit_test_world_builder.cc

```diff
@@ -3823,9 +3823,12 @@ TEST_CASE("WorldBuilder Parameters")
 {
   // First test a world builder file with a cross section defined
   std::string file = WorldBuilder::Data::WORLD_BUILDER_SOURCE_DIR + "/tests/data/type_data.json";
-  std::string file_name = WorldBuilder::Data::WORLD_BUILDER_SOURCE_DIR + "/tests/data/subducting_plate_different_angles_spherical.wb";
+  std::string file_name = WorldBuilder::Data::WORLD_BUILDER_SOURCE_DIR + "/tests/data/subducting_plate_different_angles_cartesian.wb";
```

Comment:
Aren't you abandoning the old test here?

Reply:
I noticed the only thing happening with `file_name` here is `WorldBuilder::World world(file_name);` and then `Parameters prm(world);`. Then the prm is actually initialized with a different file. These first two steps on this exact file are already being tested in line 7099-7100, so there is no reason to repeat that here. And there was no place in the unit tester which did this for the Cartesian case, so I think it is a nice expansion of the testing without losing anything. 

I could have made a new test as well, but I guess I was a bit lazy ;)